ui/src/childinfo/ChildInfo.test.js

```diff
@@ -49,40 +79,63 @@ describe('Initial Render', () => {
 
   describe('When user click next button', () => {
     beforeEach(() => {
-      childInfo = shallow(<ChildInfo name={testName} age={age}/>);
+      childInfo = shallow(<ChildInfo name={testName} age={age} />);
     })
 
-    it('Should update text on button click', () => {
+    it('Should update text on button click', async () => {
       let nextButton = childInfo.find('.next-button');
       expect(childInfo.find('.text-name').text()).toEqual(`Hi ${testName}, I have a few questions for you before we can make your wish come true!`);
-      nextButton.simulate('click');
+      await nextButton.simulate('click');
```

Comment:
Are awaits really necessary here?

Reply:
because the next button has an async call